bin/crew

```diff
@@ -193,9 +193,9 @@ def print_current_package(extra = false)
   puts ''
 end
 
-def set_package(pkg_name, pkg_path)
+def set_package(pkg_path)
```

Comment:
Maybe we can go one step further and merge `def set_package()` and `def search()`? iirc `def search()` is just a wrapper for `def set_package()` with `rescue` block

Reply:
I'm with you.  I like making things simpler.

Reply:
And we don't even need the rescue block, since it's covered by prop_test.

Reply:
I think we sometimes use it without the rescue block, but I'm not sure if there's a valid reason to do this.

Reply:
This can be the next pull request?